ui/components/ui/nft-info/nft-info.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import Box from '../box';
+import Typography from '../typography';
+import {
+  COLORS,
+  DISPLAY,
+  FONT_WEIGHT,
+  TYPOGRAPHY,
+  TEXT_ALIGN,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+import Identicon from '../identicon';
+
+export default function NftInfo({ tokenName, tokenAddress, tokenId }) {
+  const t = useContext(I18nContext);
+
+  return (
+    <Box
+      display={DISPLAY.FLEX}
+      className="nft-info"
+      backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
+    >
+      <Box
+        display={DISPLAY.FLEX}
+        justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
+        className="nft-info__content"
+      >
+        <Box margin={4}>
+          <Identicon address={tokenAddress} diameter={24} />
+        </Box>
+        <Box>
+          <Typography
+            fontWeight={FONT_WEIGHT.BOLD}
+            variant={TYPOGRAPHY.H6}
+            marginTop={4}
+            boxProps={{ display: DISPLAY.INLINE_BLOCK }}
+          >
+            {tokenName}
+          </Typography>
+          <Typography
+            variant={TYPOGRAPHY.H7}
+            display={DISPLAY.FLEX}
+            marginBottom={4}
+            color={COLORS.TEXT_ALTERNATIVE}
+          >
+            {t('tokenId')} #{tokenId}
+          </Typography>
+        </Box>
+        <Box
+          textAlign={TEXT_ALIGN.END}
+          marginTop={4}
+          marginRight={4}
+          className="nft-info__content__view"
```

Comment:
BEM class syntax doesn't allow for nesting blocks this way, if you need a block within another block then it should be named like `nft-info__content-view`

Reply:
Hi @brad-decker , You can see changes [here](https://github.com/MetaMask/metamask-extension/pull/15825/commits/a8832a59e477108990dfbe5146909100bebb7d2e#diff-8507ca3fcec29b0d76237eefb900b4688d880b82c315e4809ed9e84fc644fb41)